ofnetBgp.go

```diff
@@ -130,18 +113,26 @@ func (self *OfnetBgp) StartProtoServer(routerInfo *OfnetProtoRouterInfo) error {
 	if err != nil {
 		log.Fatal(err)
 	}
+
 	self.cc = conn
-	log.Debugf("Creating the loopback port ")
-	err = self.agent.ovsDriver.CreatePort(self.intfName, "internal", 1)
-	if err != nil {
-		log.Errorf("Error creating the port: %v", err)
+	link, err := netlink.LinkByName(self.intfName)
+	if err == nil {
+		netlink.LinkSetDown(link)
```

Comment:
This is not uplink flaps. This is internal interface link flap usually needed to clear out arps/exisiting l2 neighborship in the kernel. Typical scenrios are bgp config updates.